src/calendar/header/index.js

```diff
@@ -130,7 +130,7 @@ class CalendarHeader extends Component {
 
     return (
       <View style={this.props.style}>
-        <View style={this.style.header}>
+        {this.props.customHeader ? this.props.customHeader() : <View style={this.style.header}>
```

Comment:
I would prefer a check if 'customHeader' is even a function, or safely invoke using lodash